### ISSUE TITLE
Make it more obvious that you're logged out

### ISSUE THREAD

**Issue by chemicalcrux** [Neos-Metaverse/NeosPublic#1865]
## Is your feature request related to a problem? Please describe.
People who are new to Neos often get confused when they don't check the "Remember Me" box, then launch Neos for a second time and wind up logged out. They see an empty contacts list, missing content, etc.

It's obvious to experienced players that they are not logged in, but since the game doesn't proactively prompt for a login, it's very easy for newbies to miss this.

## Relevant issues
I did not find any.

## Describe the solution you'd like
Anything that does not work when logged out should clearly communicate the problem. For example, the contacts screen should have a big ol' "Logged out" slapped across it, because the contacts menu can't possibly work without being logged in.

## Describe alternatives you've considered
Automatically checking the "Remember me" box would help to alleviate the issue, although this has also confused people who just log in spordaically, and thus have their saved logins expire.

Prompting for a login automatically (unless you explicitly say "do not ask me to log in") would also help.

## Additional context
N/A

**Comment by shiftyscales** [comment]
There are valid use-cases for offline/accountless use of Neos, and I don't think the old UIs should be modified to account for this.

How many users are having issues with this/how pervasive a problem is it? Aside from the above hints you made not of that they aren't logged in, they would also find their cloud home doesn't load, nor their favorited avatar, on the home screen they would see that in place of their profile picture/name, they are 'Anonymous' and there is a log in button.

I feel like there are already a lot of contextual clues that can lead the user to that conclusion. On top of that, Neos always opens with the dashboard open, so the login prompt is right there in front of them.

**Comment by Frooxius** [comment]
I don't plan on updating the old UI for this, but the new UI will be designed to account for this and make it clear that you're not logged in. It's certainly one of the usability issues that we want to improve with the redesign.

**Comment by chemicalcrux** [comment]
> There are valid use-cases for offline/accountless use of Neos, and I don't think the old UIs should be modified to account for this.
> 
> How many users are having issues with this/how pervasive a problem is it? Aside from the above hints you made not of that they aren't logged in, they would also find their cloud home doesn't load, nor their favorited avatar, on the home screen they would see that in place of their profile picture/name, they are 'Anonymous' and there is a log in button.
> 
> I feel like there are already a lot of contextual clues that can lead the user to that conclusion. On top of that, Neos always opens with the dashboard open, so the login prompt is right there in front of them.

New people don't really understand what a "cloud home" is, especially since the default home world is just a differently-tinted copy of the default local world. They also often don't have an avatar favorited and don't really know their way around the menu, so they're much less likely to pick up on those cues.

They'll see that the game loaded and didn't ask them to log in -- which looks an awful lot like what'd happen if they *were* properly logged in.

This wouldn't need to be something that makes the UI inconvenient for those who are anonymous; a logged-out user isn't going to be looking at Contacts anyway.

**Comment by mralext20** [comment]
So from that it seems the problem stems from being logged out of neos after being logged in - e.g. the token expiring.

**Comment by chemicalcrux** [comment]
Yes, since it's very hard to miss the login process the first time you play.

This can happen if you don't check "Remember Me", or if you do check it but haven't been on in at least a week. 

The underlying problem is that the logged-in and logged-out states aren't glaringly distinct for inexperienced users, so they wind up trying to do something that requires you to be logged in and think that it's broken.

**Comment by FlameSoulis** [comment]
I have to very much agree with this. Unfortunately, the initial screen shown has a large amount of dumped information that a new user will not immediately catch that they aren't logged in.
![20210323213128_1](https://user-images.githubusercontent.com/6835457/112241304-5ce0a380-8c20-11eb-8f4d-5954e1f31967.png)
This could also be a terminology issue, where users may not fully understand that "Anonymous" is the same as "Logged Out."

**Comment by Frooxius** [comment]
Yeah that will be addressed with the new redesigned UI parts. The old one I don't plan expanding to include this though, as that would be a time sink.
